tests/cisco/test_cisco_switch_protocol.py

```diff
@@ -194,7 +194,7 @@ def test_show_vlan_brief(self, t):
         t.readln("")
         t.readln("VLAN Name                             Status    Ports")
         t.readln("---- -------------------------------- --------- -------------------------------")
-        t.readln("1    default                          active    Fa0/2, Fa0/3, Fa0/4")
+        t.readln("1    default                          active    Fa0/2, Fa0/3, Fa0/4, Fa0/5, Fa0/6, Fa0/7, Fa0/8, Fa0/9, Fa0/10, Fa0/11, Fa0/12")
```

Comment:
It may be a good idea for the future to always wrap at 80 as the idea is to test applications that should expect a line wrap in what the switch returns.  But I don't think this is part of this review.